doc/manual/config-advanced.rst

```diff
@@ -23,12 +23,13 @@ path of your installation) as follows:
 - *Team pictures*: a photo of the team will be shown in the team details
   page if `public/images/teams/456.jpg` exists, where *456* is the
   team's numeric ID as shown in the DOMjudge interface.
-- *Banner*: a page-wide banner can be shown on the public scoreboard
-  if that image is placed in `public/images/banner.png`.
+- *Contest Banners*: a page-wide banner can be shown on the public scoreboard
+  if that image is placed in `public/images/banners/1.png` where *1* is the
+  contest's numeric ID as shown in the DOMjudge interface.
 
 .. note::
 
-  The IDs for affiliations and teams need to be the *external ID*
```

Comment:
no team images?

Reply:
Teams do not have an external ID for some bizarre reason. I still propose we add it at some point (since now we can't do non-numeric team ID's) but until then nope.